luatest/server.lua

```diff
@@ -378,7 +379,7 @@ function Server:restart(params, opts)
     }, {wait_until_ready = '?boolean'})
 
     if not self.process then
-        log.warn("Process wasn't started")
+        log.warn("%s server process wasn't started", self.alias)
```

Comment:
`Cannot restart server %s since its process not started`

Reply:
Fixed